cmds/installcommand/installcommand.go

```diff
@@ -60,6 +67,10 @@ type form struct {
 	verbose bool
 }
 
+func r(n...string) string {
```

Comment:
Can this function name be more descriptive?

Reply:
it could but ... it's so nice and compact in all its usage. 

How about RootPath and I'll provide it in the uroot package?

Reply:
I added UrootPath to util